.github/issue_template.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Issue Report
+about: Create a report to report a bug, ask a question, or provide feedback
+title: "[BUG/QUESTION/SUGGESTION] Describe the issue in a concise title"
+labels: ''
+assignees: ''
+
+---
+
+# **Please select the type of issue you are creating:**
+
+- [ ] Bug Report
+- [ ] Question
+- [ ] Suggestion / Enhancement Request
+
+# **Description**
+
+Provide a clear and concise description of your issue or question. For Bug Reports please provide the following:
+
+## **Code to Reproduce**
+
+Provide the code snippet that demonstrates the issue. This could be a Python script, code from your application, or code used in a terminal or IDE. Be sure to include relevant function calls and parameters.
+
+``` python
+import pycountry
+
+# Your code here
+```
+
+## **Expected behavior**
+
+A clear and concise description of what you expected to happen.
+
+## **Screenshots**
+
+If applicable, add screenshots to help explain your problem.
+
+## **Environment (please complete the following information):**
+
+- Python version:
+- Operating System:
+- Any relevant dependencies and versions:
+
+# **Additional context**
+
+Add any other context or information related to your issue or question.
+
+---
+
+**Note**: Before submitting a bug report, please make sure that you have the latest version of pycountry installed and have checked for any existing similar issues. If you're reporting a security issue, please do not disclose it in a public issue. Instead, follow responsible disclosure practices and contact us privately at
+
+- [Christian Theune](mailto:ct@flyingcircus.io)
+- [Nate Schimmoller](mailto:nschimmo@gmail.com)
+- [Zachary Ware](mailto:zachary.ware@gmail.com)
```

Comment:
We could enable GitHub's security reporting instead of further publicizing our email addresses :)

Reply:
@zware how do I do that? Glad to do so :D

Reply:
@zware created a SECURITY.md file as part of the pull as well with guidance to submit via GitHub's Security Reporting

Reply:
I had clicked the (an?) "enable" button at https://github.com/pycountry/pycountry/security, your `SECURITY.md` will get us further down the path :)